core/templates/dev/head/components/RatingDisplayDirective.js

```diff
@@ -85,10 +85,9 @@ oppia.directive('ratingDisplay', [
           }
         };
         $scope.enterStar = function(starValue) {
-          if (
-              $scope.isEditable &&
-              ($scope.status === STATUS_ACTIVE ||
-                $scope.status === STATUS_INACTIVE)) {
+          var starsHaveNotBeenClicked = ($scope.status === STATUS_ACTIVE ||
```

Comment:
Move the check after `(` for the next time, so it's more readable.

Reply:
Done!